pages.tr/common/rg.md

```diff
@@ -0,0 +1,37 @@
+# rg
+
+> Ripgrep, yinelemeli satır-odaklı bir CLI arama aracıdır.
+> Grep'e daha hızlı bir alternatif olmayı hedefler. 
```

Comment:
```suggestion
> Grep'e daha hızlı bir alternatif olmayı hedefler.
```

Reply:
You're welcome and thanks for fixing it.